config/custom-environment-variables.yml

```diff
@@ -4,12 +4,12 @@ mailer:
   forgot_password_mail_template_id: 'FORGOT_PASSWORD_MAIL_TEMPLATE_ID'
 
 mongodb:
-  uri: "MONGODB_URI"
+  uri: 'MONGODB_URI'
```

Comment:
Can we please check here, and other format related changes if its issue with our boilerplate or your editor? In case its latter, we want to revert them please.